README.md

```diff
@@ -0,0 +1,146 @@
+# Capsule sandbox
+
+The Capsule sandbox is a containerized development environment for building Capsule applications in Rust.
```

Comment:
in Rust, on DPDK?

Reply:
Capsule implies there's gonna be DPDK. I will link back to the main repo from here.

Reply:
yeah, we def need some general linkage back.